tasks/runtime.yml

```diff
@@ -75,7 +80,7 @@
     group: root
     mode: 0644
   when:
-    - openhpc_enable.batch | default(false) | bool
+    - (openhpc_enable.batch | default(false) | bool) or (openhpc_enable.login | default(false) | bool)
```

Comment:
Possibly could make this check to see if slurmd is enabled instead?. That way you could just allow people to set:
```
ohpc_slurm_services:
  control: slurmctld
  batch: slurmd
  login: []
  ```
  to use SRV records?